config/coreboot-novacustom-v560tu.config

```diff
@@ -0,0 +1,961 @@
+CONFIG_SOC_INTEL_COMMON_BLOCK_USB4=y
+CONFIG_SOC_INTEL_COMMON_BLOCK_USB4_PCIE=y
+CONFIG_SOC_INTEL_COMMON_BLOCK_USB4_XHCI=y
+CONFIG_SOC_INTEL_ENABLE_USB4_PCIE_RESOURCES=y
+# CONFIG_FIRMWARE_CONNECTION_MANAGER is not set
+CONFIG_SOFTWARE_CONNECTION_MANAGER=y
+CONFIG_SOC_INTEL_COMMON_BLOCK_VTD=y
+CONFIG_ENABLE_EARLY_DMA_PROTECTION=y
+CONFIG_SOC_INTEL_COMMON_BLOCK_XDCI=y
+CONFIG_SOC_INTEL_COMMON_BLOCK_XHCI=y
+CONFIG_SOC_INTEL_COMMON_BLOCK_XHCI_ELOG=y
+
+#
+# Intel SoC Common PCH Code
+#
+CONFIG_SOC_INTEL_COMMON_PCH_CLIENT=y
+CONFIG_SOC_INTEL_COMMON_PCH_BASE=y
+CONFIG_SOC_INTEL_COMMON_PCH_LOCKDOWN=y
+CONFIG_SOC_INTEL_COMMON_SPI_LOCKDOWN_SMM=y
+CONFIG_PCH_SPECIFIC_BASE_OPTIONS=y
+CONFIG_PCH_SPECIFIC_DISCRETE_OPTIONS=y
+CONFIG_PCH_SPECIFIC_CLIENT_OPTIONS=y
+
+#
+# Intel SoC Common coreboot stages and non-IP blocks
+#
+CONFIG_SOC_INTEL_COMMON_BASECODE=y
+CONFIG_SOC_INTEL_COMMON_RESET=y
+CONFIG_SOC_INTEL_COMMON_ACPI_WAKE_SOURCE=y
+CONFIG_PAVP=y
+# CONFIG_SOC_INTEL_DEBUG_CONSENT is not set
+CONFIG_HAVE_INTEL_COMPLIANCE_TEST_MODE=y
+# CONFIG_SOC_INTEL_COMPLIANCE_TEST_MODE is not set
+CONFIG_SOC_INTEL_CRASHLOG=y
+CONFIG_SOC_INTEL_IOE_DIE_SUPPORT=y
+
+#
+# CPU
+#
+CONFIG_CPU_INTEL_FIRMWARE_INTERFACE_TABLE=y
+CONFIG_CPU_INTEL_COMMON=y
+CONFIG_ENABLE_VMX=y
+CONFIG_SET_IA32_FC_LOCK_BIT=y
+CONFIG_SET_MSR_AESNI_LOCK_BIT=y
+CONFIG_CPU_INTEL_COMMON_VOLTAGE=y
+CONFIG_CPU_INTEL_COMMON_SMM=y
+CONFIG_PARALLEL_MP=y
+CONFIG_PARALLEL_MP_AP_WORK=y
+CONFIG_DEFAULT_X2APIC_LATE_WORKAROUND=y
+# CONFIG_XAPIC_ONLY is not set
+# CONFIG_X2APIC_ONLY is not set
+# CONFIG_X2APIC_RUNTIME is not set
+CONFIG_X2APIC_LATE_WORKAROUND=y
+CONFIG_UDELAY_TSC=y
+CONFIG_TSC_MONOTONIC_TIMER=y
+CONFIG_X86_CLFLUSH_CAR=y
+CONFIG_HAVE_SMI_HANDLER=y
+CONFIG_SMM_TSEG=y
+CONFIG_SMM_PCI_RESOURCE_STORE_NUM_SLOTS=8
+CONFIG_X86_INIT_NEED_1_SIPI=y
+CONFIG_AP_STACK_SIZE=0x800
+CONFIG_SMP=y
+CONFIG_SSE=y
+CONFIG_SSE2=y
+CONFIG_SUPPORT_CPU_UCODE_IN_CBFS=y
+CONFIG_USE_CPU_MICROCODE_CBFS_BINS=y
+CONFIG_CPU_MICROCODE_CBFS_DEFAULT_BINS=y
+# CONFIG_CPU_MICROCODE_CBFS_EXTERNAL_BINS is not set
+# CONFIG_CPU_MICROCODE_CBFS_EXTERNAL_HEADER is not set
+# CONFIG_CPU_MICROCODE_CBFS_NONE is not set
+
+#
+# Northbridge
+#
+
+#
+# Southbridge
+#
+CONFIG_PCIEXP_HOTPLUG=y
+CONFIG_INTEL_DESCRIPTOR_MODE_REQUIRED=y
+CONFIG_SOUTHBRIDGE_INTEL_COMMON_SMBUS=y
+CONFIG_SOUTHBRIDGE_INTEL_COMMON_PIRQ_ACPI_GEN=y
+CONFIG_HAVE_INTEL_CHIPSET_LOCKDOWN=y
+CONFIG_INTEL_DESCRIPTOR_MODE_CAPABLE=y
+# CONFIG_VALIDATE_INTEL_DESCRIPTOR is not set
+# CONFIG_INTEL_CHIPSET_LOCKDOWN is not set
+CONFIG_FIXED_RCBA_MMIO_BASE=0xfed1c000
+CONFIG_RCBA_LENGTH=0x4000
+
+#
+# Super I/O
+#
+
+#
+# Embedded Controllers
+#
+CONFIG_EC_ACPI=y
+CONFIG_EC_SYSTEM76_EC=y
+# CONFIG_EC_SYSTEM76_EC_UPDATE is not set
+
+#
+# Intel Firmware
+#
+CONFIG_IFDTOOL_DISABLE_ME=y
+CONFIG_HAVE_ME_BIN=y
+# CONFIG_STITCH_ME_BIN is not set
+# CONFIG_ME_REGION_ALLOW_CPU_READ_ACCESS is not set
+CONFIG_HAVE_INTEL_ME_HAP=y
+# CONFIG_INTEL_ME_DISABLED_HECI is not set
```

Comment:
HAP disabled?

Reply:
with ME disabled, S0ix will stop working correctly (CPU won't enter deepest sleepstates and fans will stay on), and there's no S3, so I think it has to stay enabled for now.

Reply:
@mkopec confused: https://github.com/linuxboot/heads/issues/1801#issue-2557364874

Reply:
@mkopec updates on this? That would be the first Heads platform with ME enabled, that was not expected/planned https://github.com/linuxboot/heads/issues/1801

Reply:
I'm not sure what happened here, summoning @macpijan for help

Reply:
Indeed, we have supported/tested S0ix only in UEFI firmware, and UEFI firmware, which was the reference here. I think this is now clarified: https://github.com/linuxboot/heads/pull/1871#issuecomment-2532080947

I hope this thread can be resolved once @mkopec changes these settings